src/utils/flagCodes.ts

```diff
@@ -8,7 +8,12 @@ export interface Flag<T extends string> {
 
 // OUR specific flags
 // lowercase hex value to be appended to tx.data
-const SENTINEL = 'dec0de'
+const SENTINEL = process.env.SENTINEL || CONFIG.transactions.sentinel
+console.log('SENTINEL', SENTINEL)
+
+if (!/^[0-9a-f]+$/.test(SENTINEL)) {
+  throw new Error(`SENTINEL isn't valid. Expected lowercase hex value, got ${SENTINEL}`)
```

Comment:
Shouldn't we validate that SENTINEL is 6 characters?

Reply:
It doesn't have to be 6 characters. Do we mention 6 somewhere?

Reply:
One question. When we consume this information, and in case there are sentinels from multiple forks. How do we index that info too?
Since I was in the assumption that they had fixed number of chars, my thinking was take the last 12 chars, where 6 are the analytics and 6 are the sentinel.

Can the sentinel be 0, or 00000?

I understand that knowing the ABI we can interpret the hex in the data, and know if there's additional data, but I'm not so sure about how easy is that in dune, if for example the call can have a variable size input (like an array).